mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +26,41 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Should image open?
+   */
+  isShouldOpen?: boolean;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
+
+  const [isOpen, setIsOpen] = useState(false);
+
   const className = clsx(styles.image, props.className);
 
+  /**
+   * Function full screen image
+   */
+  const onClickHandler = (value: boolean) => {
+    setIsOpen(value);
+  };
+
   return (
-    <img
-      src={props.src}
-      alt={props.alt}
-      className={className}
-      data-cy={props.dataCy}
-    />
+    <div className={isOpen ? styles.imageWrapperFullScreen : ""}>
```

Comment:
```suggestion
    <div className={isOpen ?? styles.imageWrapperFullScreen}>
```

Reply:
ok

Reply:
Yes? sory, I was mean
isOpen && styles.imageWrapperFullScreen

Reply:
@Ekaterina1994 if "isOpen" will be false then className={false}

Reply:
The provided code snippet uses a conditional statement to apply a class based on the value of isOpen. Here's how it works:

className: This is a JavaScript property used to set the class names for an HTML element.
{isOpen && styles.imageWrapperFullScreen}: This is a JavaScript expression within curly braces.
isOpen: This is a boolean variable.
&&: This is the logical AND operator.
styles.imageWrapperFullScreen: This assumes styles is an object containing CSS class names. Here, it accesses the imageWrapperFullScreen class from the styles object.
If isOpen is true:

The entire expression evaluates to true because true && anything is always true.
Therefore, the class name applied will be styles.imageWrapperFullScreen.
If isOpen is false:

The expression isOpen && styles.imageWrapperFullScreen evaluates to false because false && anything is always false.
Since the entire expression evaluates to false, no additional class is applied to the element's className property. The element will only have the base classes defined outside the curly braces.
In summary:

When isOpen is false, the element won't have the styles.imageWrapperFullScreen class applied. It will retain any other classes defined for it outside the conditional statement.

Reply:
let b = 0 && 1

Reply:
@Ekaterina1994 let b = 0 && 1

Reply:
@Ekaterina1994 The && operator returns the last argument if everything is true or the first value is false. In this case.
For example let b = 0 && 1. b will be 0